packages/xo-server-backup-reports/src/index.js

```diff
@@ -245,6 +245,8 @@ class BackupReportsXoPlugin {
       if (
         !force &&
         (reportWhen === 'never' ||
+          // Introduced by: https://github.com/vatesfr/xen-orchestra/pull/2869/commits/753ee994f2948bbaca9d3161eaab82329a682773#diff-9c044ab8a42ed6576ea927a64c1ec3ebR105
```

Comment:
- "Handle improper value introduced by"
- `/commit/` instead of `/pull/2869/commits/`?